docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -6,22 +6,177 @@ dev_langs:
+
+You can see the syntax for *switch expressions* in the following example:
+
+```csharp
+static string Display(object o) => o switch
+{
+    Point { X: 0, Y: 0 }         => "origin",
+    Point { X: var x, Y: var y } => $"({x}, {y})",
+    _                            => "unknown"
+};
+```
+
+There are two patterns at play in this example. `o` first matches with the `Point` *type pattern* and then with the *property pattern* inside the *{curly braces}*. The `_` describes the `discard pattern`, which is the same as `default` for *switch statements*.
+
+All of these patterns enable you to write declarative code that captures your intent instead of procedural code that implements tests for it. The compiler becomes responsible for implementing that boring procedural code and is guaranteed to always do it correctly.
+
+There will still be cases where *switch statements* will be a better choice than *switch expressions* and patterns can be used with both syntax styles.
+
+For more information, see [Do more with patterns in C# 8.0](https://blogs.msdn.microsoft.com/dotnet/2019/01/24/do-more-with-patterns-in-c-8-0/).
+
+### Pointers to Unmanaged Constructed Types
+
+You can now take a pointer to [unmanaged constructed types](https://github.com/dotnet/csharplang/issues/1744), such as `ValueTuple<int, int>`, as long as all of the elements of the generic type are unmanaged. Until now, you could only take a pointer to non-generic structs with struct-fields. Given the prevalence of generic structs in more recent .NET Core releases, this change is critical to enable performant code for common types like `ValueTask<T>`, `ValueTuple<T>` and `Span<T>`.
+
+The following is an example of an opportunity in the platform where this feature is used:
+
+**Before:** [System/Runtime/Intrinsics/Vector256.cs#L1303](https://github.com/dotnet/coreclr/blob/57fd77e6f8f7f2c37cc5c3b36df3ea4f302e143b/src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector256.cs#L1303)
+
+```csharp
+Vector256<short> SoftwareFallback(short x)
+{
+    var result = Vector256<short>.Zero;
+    Unsafe.WriteUnaligned(ref Unsafe.As<Vector256<short>, byte>(ref result), value);
+    return result;
+}
+```
+
+**After:**
+
+```csharp
+Vector256<int> SoftwareFallback(int x)
+{
+    var result = Vector256<int>.Zero;
+    ((int*)(&result))[0] = x;
+    return result;
+}
+```
+
+## IEEE Floating-point improvements
+
+Floating point APIs are in the process of being updated to comply with [IEEE 754-2008 revision](https://en.wikipedia.org/wiki/IEEE_754-2008_revision). The goal of [this floating point project](https://github.com/dotnet/corefx/issues/31901) is to expose all "required" operations and ensure that they are behaviorally compliant with the IEEE spec.
```

Comment:
We should probably be more conservative with GH links in our official docs and leave that to blog posts. I think most readers will not get value from those GH links.